prime-angular-frontend/src/app/shared/components/dialogs/content/claim-enrollee/claim-enrollee.component.html

```diff
@@ -1,17 +1,25 @@
 <h2 mat-dialog-title
     class="d-flex align-items-stretch">
   <span class="flex-grow-1">
-    Disclaim Enrollee
+    {{ this.getTitle() }}
   </span>
 </h2>
 <mat-dialog-content>
-  <span>You can unclaim the enrollee or assign them to a colleague.</span>
+  <span>{{ this.getPromptText() }}</span>
 
   <div class="mt-4">
-    <button mat-flat-button
-            color="primary"
-            (click)="onDisclaim()"
-            class="mr-4">Disclaim Enrollee</button>
+    <span *ngIf="displayAsClaimPrompt()">
+      <button mat-flat-button
+              color="primary"
+              (click)="onClaim()"
+              class="mr-4">Claim</button>
+    </span>
+    <span *ngIf="!displayAsClaimPrompt()">
```

Comment:
You'll want to use a `templateRef` that's referenced in the above `*ngIf` using an `else`-statement, and change it to use a `ng-container` instead of a `<span>`.  If we can avoid adding markup to the template we should to decrease complexity.  `ng-container` is a placement marker and when rendered it will render it's content or the else content underneath the tag so it doesn't disrupt the hierarchy of the styles.

```
<ng-container *ngIf="possibleAction === ClaimActionEnum.Claim; else disclaim">

// Removed for brevity

<button #disclaim 
        mat-flat-button ...>
```